plugins/catalog-react/src/components/UserListPicker/UserListPicker.tsx

```diff
@@ -122,17 +128,30 @@ export const UserListPicker = ({
   const classes = useStyles();
   const configApi = useApi(configApiRef);
   const orgName = configApi.getOptionalString('organization.name') ?? 'Company';
-
   // Remove group items that aren't in availableFilters and exclude
   // any now-empty groups.
-  const filterGroups = getFilterGroups(orgName)
+  const initialFilterGroup = getFilterGroups(orgName)
     .map(filterGroup => ({
       ...filterGroup,
       items: filterGroup.items.filter(
         ({ id }) => !availableFilters || availableFilters.includes(id),
       ),
     }))
     .filter(({ items }) => !!items.length);
+  const [filterGroups, setFilterGroups] =
+    useState<ButtonGroup[]>(initialFilterGroup);
+  const setDefaultFilterGroups = useCallback(() => {
```

Comment:
Seems a little bit strange to have this defined outside as a callback tbh? Would it make sense to just inline this in the useEffect cleanup?